KK-1372 | Fix type generating, upgrade all except HDS/eslint/react, fix react-transition-group related warning

karisal-anders opened the pull request:
## Description

- Fix type generating
- upgrade all except HDS/eslint/react
- fix react-transition-group related warning by replacing it with react-toastify
- for accessibility don't auto-close toast notifications

## Context

[KK-1372](https://helsinkisolutionoffice.atlassian.net/browse/KK-1372)

## How Has This Been Tested?

<!-- Explain what automated and manual testing approaches you have used -->

## Manual Testing Instructions for Reviewers

<!-- Make it easy for reviewers to test your changes by providing instructions -->

## Screenshots

<!-- Add screenshots if appropriate -->


[KK-1372]: https://helsinkisolutionoffice.atlassian.net/browse/KK-1372?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

terovirtanen commented:
**KUKKUU-UI branch is deployed to platta: https://kukkuu-ui-pr620.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
## TestCafe result is __success__ for https://kukkuu-ui-pr620.dev.hel.ninja :laughing::tada::tada::tada:

terovirtanen commented:
**KUKKUU-UI branch is deployed to platta: https://kukkuu-ui-pr620.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
## TestCafe result is __success__ for https://kukkuu-ui-pr620.dev.hel.ninja :laughing::tada::tada::tada:

terovirtanen commented:
**KUKKUU-UI branch is deployed to platta: https://kukkuu-ui-pr620.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
## TestCafe result is __success__ for https://kukkuu-ui-pr620.dev.hel.ninja :laughing::tada::tada::tada:

terovirtanen commented:
**KUKKUU-UI branch is deployed to platta: https://kukkuu-ui-pr620.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
## TestCafe result is __success__ for https://kukkuu-ui-pr620.dev.hel.ninja :laughing::tada::tada::tada:

karisal-anders commented:
Withdrew code review requests, needs more testing.

karisal-anders commented:
Force pushed with changes:
- Rebased on current master
- Updated the commit that updated all packages to latest minor versions with current latest minor versions
- Fixed the "Success enrolling" toast to look ok, previously it looked wrong
  - This is not exactly like it was before, but at least usable

terovirtanen commented:
**KUKKUU-UI branch is deployed to platta: https://kukkuu-ui-pr620.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
## TestCafe result is __success__ for https://kukkuu-ui-pr620.dev.hel.ninja :laughing::tada::tada::tada:

terovirtanen commented:
**KUKKUU-UI branch is deployed to platta: https://kukkuu-ui-pr620.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
## TestCafe result is __success__ for https://kukkuu-ui-pr620.dev.hel.ninja :laughing::tada::tada::tada:
